torch/_inductor/compile_fx.py

```diff
@@ -372,7 +372,13 @@ def compiled_artifact(new_inputs):
                         "skipping cudagraphs due to multiple device indexes"
                     )
 
-    result = align_inputs(compiled_graph, example_inputs, range(num_fixed))
+    # cudagraphs does its own aligning of inputs
+    if not cudagraphs:
+        new_callable = align_inputs(
```

Comment:
Nit: we could always assign to `compiled_graph.current_callable` without doing the checking next.

Reply:
true, will add on in follow up since i have tests passing..